client/src/parsec/functions.ts

```diff
@@ -59,7 +59,7 @@ export async function listAvailableDevices(): Promise<Array<AvailableDevice>> {
   return await libparsec.listAvailableDevices(window.getConfigDir());
 }
 
-function getClientConfig(): ClientConfig {
+export function getClientConfig(): ClientConfig {
```

Comment:
This function should be put into another file, something like `internals.ts` that isn't exported by the index.ts. This way the `parsec` module can do `import { getClientConfig } from '@/parsec/internals';` but a dev cannot do `import { getClientConfig } from '@/parsec';`. This is also true for the `_wait` function, and for the `MOCK_HOST_WAITING_TIME` variable that could be changed into `MOCK_WAITING_TIME`.

This can be done later after this PR has been merged.